src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/FlowMonitoringTopology.java

```diff
@@ -120,15 +122,23 @@ private void tickBolt(TopologyBuilder topologyBuilder) {
                 ComponentId.TICK_BOLT.name());
     }
 
+    private void flowStateCacheBolt(TopologyBuilder topologyBuilder, PersistenceManager persistenceManager) {
+        FlowStateCacheBolt flowStateCacheBolt = new FlowStateCacheBolt(persistenceManager, ZooKeeperSpout.SPOUT_ID);
+        declareBolt(topologyBuilder, flowStateCacheBolt, ComponentId.FLOW_STATE_CACHE_BOLT.name())
+                .allGrouping(ComponentId.FLOW_SPLITTER_BOLT.name(), FLOW_UPDATE_STREAM_ID.name())
+                .allGrouping(ComponentId.TICK_BOLT.name())
+                .allGrouping(ZooKeeperSpout.SPOUT_ID);
```

Comment:
you receive commands from ZooKeeperSpout but do not sent responses to ZookeeperBolt. 
Please add stream from ZooKeeperSpout to ZookeeperBolt. Like in FlowCacheBolt

Reply:
Fixed